tools/commands-packages-query.js

```diff
@@ -62,6 +63,44 @@ var formatDependencies = function (versionDependencies) {
   return dependencies;
 };
 
+// Convert package exports into a more helpful format for output in
+// 'meteor show'.
+var formatExports = function (rawexports) {
```

Comment:
Hm, that seemed harder to understand, to me, than just adding helper functions (and overengineering), but I guess I see your point and will change it.